app/questionnaire/navigator.py

```diff
@@ -16,10 +16,15 @@ def evaluate_rule(when, answer_value):
     match_value = when['value']
     condition = when['condition']
 
+    answer_to_test = answer_value
+
+    if isinstance(answer_value, list) and len(answer_value) == 1:
```

Comment:
If there are more than one answer should we not test for any of them to match the goto rule?